common/resource_manager.cc

```diff
@@ -243,7 +243,7 @@ std::unique_ptr<ResourceManager::Resource> ResourceManager::GetMatchedResource(
   if (!app_control_info.src().empty()) {
     return std::unique_ptr<Resource>(new Resource(
       InsertPrefixPath(app_control_info.src()),
-                       app_control_info.onreset() == "disable" ? false : true));
+                       app_control_info.reload() == "disable" ? false : true));
```

Comment:
It already changed by 

commit 281d5a86720db88bda1af1beeb6ff076d7bef759
Author: psikorski <psikorski@users.noreply.github.com>
Date:   Mon Oct 19 11:09:04 2015 +0200

    Revert "Revert "config.xml parameter change: on-reset -> reload""